### ISSUE TITLE
Create a subtab under "Science Enabled" for public user profiles for collected holotypes

### ISSUE THREAD

**Issue by dshorthouse** [bionomia/bionomia#393]
As in the subject. Include images and DOIs to original descriptions when known and present.

**Comment by dshorthouse** [comment]
@rdmpage - have you compiled a mapping between gbifIDs and DOIs for taxonomic works that referenced types akin to the work you did on [JSOR](https://iphylo.blogspot.com/2023/10/where-are-plant-type-specimens-mapping.html)? If so, is it accessible somewhere to draw into Bionomia + whatever attribution would be nice to show?

**Comment by rdmpage** [comment]
@dshorthouse 

### TL;DR

I think these datasets which are in ChecklistBank maybe what you are after:

- https://www.checklistbank.org/dataset/128415/about
- https://www.checklistbank.org/dataset/164203/about
- https://www.checklistbank.org/dataset/129659/about

They link new taxonomic names to DOIs and Wikidata (and are subsets of larger, ongoing obsessive efforts to link all names to whatever bibliographic identifiers I can find). Feel free to use, citing the DOIs for the datasets would be nice (the DOIs resolve to Zenodo).

### Longer version 

To make sure I understand what you’re after, my simple model is this:

```mermaid
graph TD
    w(work) --> |cites| s(specimen)
    %% w --> |doi|doi
    t(taxon name) --> |hasType|s
    t --> |published in|w
```

Presumably you want this:

```mermaid
graph TD
    w(work) --> |cites| s(specimen)
    %%w --> |doi|doi
    %%t(taxon name) --> |hasType|s
    %%t --> |published in|w
```

Ideally we would have this directly. Plazi sort of does this but invents new GBIF records for each specimen rather than linking to existing records (argh). I spent a little time [some years ago](https://iphylo.blogspot.com/2012/02/gbif-specimens-in-biostor-who-are-top.html) linking BHL content to specimens based on finding specimen codes in text. This would be fun to revive.

In the absence of direct evidence of citation (I.e., specimen code in text) I guess you want this:

```mermaid
graph TD
    w(work) --> |cites| s(specimen)
    %%w --> |doi|doi
    %%t(taxon name) --> |hasType|s
    t(taxon name) --> |published in|w
```

So the citation is inferred from the fact that a work published a new name, that name has a type specimen, ergo the publication must have cited the type.

For this you need links between first publication of names and corresponding taxon names, and that is what [BioNames](https://bionames.org) and related projects do. Subsets of these (focussed on works with DOIs and/or Wikidata ids) are available in ChecklistBank and Zenodo (see above). If these are useful feel free to grab the data.  Maybe cite the relevant dataset DOI for each assertion as this would provide provenance so users know who to blame for my inevitable errors?

Of course, this presupposes that GBIF records include the original name associated with the type specimen, this is not always the case. Ironically a good signal that the original name IS associated with the specimen is if there's not a match with the GBIF taxonomy.

Hope this helps...

**Comment by dshorthouse** [comment]
> Of course, this presupposes that GBIF records include the original name associated with the type specimen, this is not always the case. Ironically a good signal that the original name IS associated with the specimen is if there's not a match with the GBIF taxonomy.

This is a kicker. I naively assumed that most data publishers used the published name in `dwc:scientificName` for `dwc:typeStatus` = 'holotype' (I'd not use the inferred name in GBIF through whatever reconciliation has been done via their nub classification). And so, what I had in mind is significantly more pedestrian. Given that that there are ~500k such occurrence records in GBIF that do have `dwc:typeStatus` = 'holotype', my hope was indeed the reverse relationship in your second figure above whereby `specimen` is `citedBy` a `work`, which of course presupposes someone has done the hard work to draw the relationships in your first figure.

**Comment by rdmpage** [comment]
OK, just to be clear. Consider https://www.gbif.org/occurrence/3355827325 which is the holotype of _Macrogomphus phalantus jayavarman_ recorded by `O. Kosterin`. I search in BioNames and I have this record http://bionames.org/search/Macrogomphus+phalantus+jayavarman, 

![Screenshot 2024-12-13 at 08 38 41](https://github.com/user-attachments/assets/c5e3371c-a608-4cfd-9cba-f8d7b297a766)

So we have _Macrogomphus phalantus jayavarman_ `urn:lsid:organismnames.com:name:5452780`published in https://dx.doi.org/10.11646/zootaxa.4615.1.3, i.e.:

## 1

```mermaid
graph TD
    %%w(doi:10.11646/zootaxa.4615.1.3) --> |cites| s(gbif:3355827325)
    %%t(taxon name) --> |hasType|s
    t(Macrogomphus plants jayavarman) --> |published in|w(doi:10.11646/zootaxa.4615.1.3)
```

We could assume that this means:


## 2

```mermaid
graph TD
    w(doi:10.11646/zootaxa.4615.1.3) --> |cites| s(gbif:3355827325)
    %%t(taxon name) --> |hasType|s
    %%t(Macrogomphus plants jayavarman) --> |published in|w(doi:10.11646/zootaxa.4615.1.3)
```

I can give you lots of (1) if you are happy to make assumption (2).
